src/collatzConjecture.js

```diff
@@ -0,0 +1,15 @@
+export default function collatzConjecture(n){
+  let collatzArr = []
```

Comment:
I encourage you to avoid the use of abbreviations in your variable names - this may make it harder for others to understand the intent of the variable.  Consider the use of a more intention revealing name (`result`).